packages/dm-core-plugins/src/explorer/components/Sidebar.tsx

```diff
@@ -8,8 +9,9 @@ type TProps = {
 
 export default function Sidebar(props: TProps) {
   const { children } = props
+  const sideBarWidth = useLocalStorage('explorerWidth', defaultWidth)
```

Comment:
This does not seem to be used correctly. 
You can use it like a normal "useState"-hook. It returns  [ value, setValue ].

Meaning you can replace your useState hook with this.

Reply:
Can't get it to work... The value seems to be always writing the same.